docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
     - rqworker
   nginx:
     command: nginx -c /etc/netbox-nginx/nginx.conf
-    image: nginx:1.15-alpine
+    image: nginx:1.17.6-alpine
```

Comment:
Wouldn't it suffice to put `1.17-alpine`? This way we would benefit automatically from minor version updates.

Reply:
Yes, i change it